### ISSUE TITLE
WYSIWYG editor (issues, wiki) "revert to plaintext" action seems irreversible

### ISSUE THREAD

**Issue by k9t9** [go-gitea/gitea#14453]
The issue was tested on the official demo @ https://try.gitea.io

- Gitea version (or commit ref): 1.14.0+dev-605-gd2ea21d0d  
 
## Tl;DR;

The `"revert to plaintext mode"' action in WYSIWYG editor mode ( issues, wiki ) seems to be irreversible from a UX POV.

## Issue Description

While writing / editing an issue with the WYSIWYG editor there is an icon labeled `"Revert to simple text area"` ( screenshot No.1 below) 

### Screenshot No.1
![2021-01-25_122956](https://user-images.githubusercontent.com/23067654/105662067-d2bedc00-5f09-11eb-9435-00ccf5638a29.png)

### Expected behavior:

The expected behavior is to be able to switch back when needed - while currently  Clicking the icon will result in hiding the toolbar.

Simply put - there seems to be no apparent way of turning the WYSIWYG toolbar back in case of a user mistake or need - see screenshot No.2 below.

### Screenshot No.2

![2021-01-25_123040](https://user-images.githubusercontent.com/23067654/105662092-df433480-5f09-11eb-8612-b674b044e9b5.png)

## Proposed solution : 

Since the toolbar is hidden maybe create another menu item ( menu screenshot below ) to revert back to WYSIWYG mode - or move the whole function of reverting to this menu itself ( with or without icon ) 

### Screenshot No.3

![2021-01-25_123103](https://user-images.githubusercontent.com/23067654/105662187-10bc0000-5f0a-11eb-931d-3310c3f3aac1.png)

**Comment by zeripath** [comment]
This should not hold up 1.14 - moving this to 1.15

**Comment by wxiaoguang** [comment]
The editor has been fundamentally improved in 1.20, welcome to try.
